matchzoo/models/cdssm_model.py

```diff
@@ -1,29 +1,31 @@
 """An implementation of CDSSM (CLSM) model."""
+import typing
 
 from matchzoo import engine
+from matchzoo import tasks
+from matchzoo import TensorType
+from matchzoo import preprocessors
 
 from keras.models import Model
-from keras.layers import Input, Conv1D, GlobalMaxPool1D, Dot
+from keras.layers import Input, Conv1D, Dense, Dot
```

Comment:
Our convention is to `import keras` then use `keras.layers.Conv1D`, `keras.layers.Dense`, e.t.c. because `matchzoo.layers` also exist and these two should not be confused.